app/views/record/_availability.html.erb

```diff
@@ -2,6 +2,8 @@
 <% if @record.fulltext_link.present? %>
   <% if check_online? %>
```

Comment:
Thinking through what happens here...if they're not authenticated, when they click through to SFX, they're going to face an authentication check in there if required, right?

Is that what we want? Or do we want to check if they're guests *first* and always display "sign in for full text link", unless it's an OA resource?

(Did we just secretly solve the OA badging question?)

Reply:
Maybe what I am asking here is, has UX seen this logic?

Reply:
The added logic I put in here involves the time bombed pdf links. We _can't_ show those to people that are guests as they always return the full content with no security checks on the other end (argh!). This looks for signs of a pdflink and won't show it to guests.